metallb/Makefile

```diff
@@ -0,0 +1,21 @@
+.PHONY: config_metallb start_frr
```

Comment:
need to add the rest of the targets in this Makefile to this list

Reply:
added.